src/components/HeaderSearch.vue

```diff
@@ -180,22 +146,38 @@ export default defineComponent({
         return;
       }
 
-      if (optionsRaw.length === 0) {
-        if (isValidHex(inputValue.value) && inputValue.value.length == 64) {
-          await router.push({
-            name: 'transaction',
-            params: { transaction: inputValue.value }
-          });
-          router.go(0);
-        } else {
-          await router.push({
-            name: 'account',
-            params: {
-              account: inputValue.value
-            }
-          });
-          router.go(0);
-        }
+      if (isValidHex(inputValue.value) && inputValue.value.length == 64) {
+        await router.push({
+          name: 'transaction',
+          params: { transaction: inputValue.value }
+        });
+        router.go(0);
+      } else if (
+        inputValue.value.length == 53 &&
+        inputValue.value.startsWith('EOS')
+      ) {
+        await router.push({
+          name: 'key',
+          params: { key: inputValue.value }
+        });
+        router.go(0);
+      } else if (
+        inputValue.value.length == 57 &&
+        inputValue.value.startsWith('PUB_K1')
```

Comment:
this condition and the one on line 155 both push the same route & params, could they be combined?